src/main/java/bio/terra/cli/command/notebooks/Create.java

```diff
@@ -0,0 +1,108 @@
+package bio.terra.cli.command.notebooks;
+
+import bio.terra.cli.app.AuthenticationManager;
+import bio.terra.cli.app.DockerToolsManager;
+import bio.terra.cli.model.GlobalContext;
+import bio.terra.cli.model.TerraUser;
+import bio.terra.cli.model.WorkspaceContext;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+
+/** This class corresponds to the third-level "terra notebook create" command. */
+@CommandLine.Command(
+    name = "create",
+    description = "Create a new AI Notebook instance within your workspace.")
+public class Create implements Callable<Integer> {
+
+  @CommandLine.Parameters(
+      index = "0",
```

Comment:
I got a little caught up by this when testing locally, as it was easy to miss the <instanceName> entry in the CLI help screen, since it gets tacked on after all the optional arguments:

<img width="572" alt="Screen Shot 2021-02-08 at 10 48 36 AM" src="https://user-images.githubusercontent.com/51842/107243684-40373600-69fb-11eb-9678-c4aad26c95af.png">

Form a user perspective, I don't think there's a super compelling reason to want to name my own notebook every time. I'd be happy if Terra chose a reasonable name (maybe a sanitized version of my email address?) by default, but let me override.

Easy follow-up PR if you agree with that direction.

Reply:
It's a good question. I'll check in with Nicole to get her take.